dev/com.ibm.ws.webcontainer.security/src/com/ibm/ws/webcontainer/security/WebAppSecurityConfig.java

```diff
@@ -155,6 +155,8 @@ public interface WebAppSecurityConfig {
 
     String getSameSiteCookie();
 
+    boolean getPartitionedCookie();
```

Comment:
```suggestion
    boolean isPartitionedCookie();
```

Reply:
I'll make the method name change in another PR